README.md

```diff
@@ -89,23 +90,26 @@ The following table shows how the properties from the Unified Catalog object are
 
 ### Subscription handling
 
-Unified Catalog subscription and Marketplace are different.
-For Unified Catalog there is one subscription per application per Catalog Item
-From the Marketplace it will be translated to one subscription per plan and per application to guarantee that the provider can enforce the quota correct, that is defined in the product plan.
+Unified Catalog subscriptions and Marketplace subscriptions are different.
+
+For the Unified Catalog, there is one subscription per application per Catalog Item.
+
+From the Marketplace this will be translated into one subscription per plan and application to guarantee that the provider can correctly enforce the quota that is defined in the product plan.
 
 ![Alt text](subscription.png)
 
-Due to that, the migration script creates only 1 subscription for each individual resources available in the product. The subscription is reused to add the Access to the various application.
+Due to this difference, the migration script creates only 1 subscription for each resource available in the product. The subscription is reused to add access to the various applications.
 
 **Limitation**:
-On Marketplace side, it is not possible to access the same product resource using a subscription plan having a quota restriction and multiple applications, to help provider to correctly enforce the subscription plan quotas.
-The migration script displays this message `/!\ Cannot add access to {UNIFED_CATAOLOGSUBSCRIPTION_APPLICATION_NAME} using subscription {MARKETPLACE_SUBSCRIPTION_NAME} : access already exist for another application` if multiple applications try to access the same resource using the same subscription.
+On the Marketplace side, it is not possible to access the same product resource using a subscription plan having a quota restriction and multiple applications. This is to help providers correctly enforce the subscription plan quotas.
+
+The migration script displays this message `/!\ Cannot add access to {UNIFED_CATAOLOGSUBSCRIPTION_APPLICATION_NAME} using subscription {MARKETPLACE_SUBSCRIPTION_NAME}: access already exist for another application` if multiple applications try to access the same resource using the same subscription.
 
 You can overcome this by using the unlimited quota variable: `PLAN_QUOTA="unlimited"`
 
 ## Usage
 
-Migrating all catalog item link to an environment:
+Migrating all catalog item links to an environment:
```

Comment:
@ingo for that one, what I wanted to say was: Migrating all catalog item that are part of an environment.
To me "links" should be replaced by "linked" to reflect my original thinking.

Reply:
Agreed - linked is the correct word